pkg/nuclide-external-interfaces/1.0/atom.js

```diff
@@ -660,6 +661,7 @@ declare class atom$TextEditor extends atom$Model {
   id: number,
   firstVisibleScreenRow: number,
   rowsPerPage: number,
+  element: atom$TextEditorElement,
```

Comment:
Also a private API. You shouldn't need this. Instead, use `atom.views.getView(editor)`